core-web/libs/sdk/angular/src/lib/resolver/dotcms-page.resolver.ts

```diff
@@ -29,18 +29,17 @@ export const DotCMSPageResolver: ResolveFn<
         language_id: queryParams['language_id'],
         mode: queryParams['mode'],
         variantName: queryParams['variantName'],
-        'com.dotmarketing.persona.id':
-        queryParams['com.dotmarketing.persona.id'] || '',
+        'com.dotmarketing.persona.id': queryParams['com.dotmarketing.persona.id'] || ''
     };
 
     const navProps = {
         path: '/',
         depth: 2,
-        languageId: queryParams['language_id'],
+        languageId: queryParams['language_id']
     };
 
     const pageRequest = client.page.get(pageProps) as Promise<{ entity: DotCMSPageAsset }>;
-    const navRequest = client.nav.get(navProps) as Promise<{ entity: DotCMSNavigationItem }>;
+    const navRequest = client.nav.get(navProps) as Promise<{ entity: DotcmsNavigationItem }>;
```

Comment:
This is a HUUUUUUUGE no, the request of the page can't be inside the SDK never.

The data of the page should be passed from the angular app into the component.

Reply:
Now the resolves is in angular example